src/turplanlegger/routers/notes.py

```diff
@@ -0,0 +1,59 @@
+from http import HTTPStatus
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, HTTPException, Response
+from sqlmodel import Session
+
+from ..sql import crud
+from ..sql.database import get_session
+from ..sql.models import NoteCreate, NoteRead, NoteUpdate
+
+router = APIRouter(
+    tags=['notes'],
+    prefix='/notes',
+    responses={404: {'description': 'Not found'}},
+)
+
+
+@router.get('/', description='Get all notes', response_model=list[NoteRead])
+def all_notes(session: Session = Depends(get_session)):
+    db_notes = crud.get_all_notes(session)
```

Comment:
Ser ut som denne henter ut alle notes nå, burde jeg få lov til å hente ut dine notes?

Reply:
Ja, har ikke implementert rettigheter, i Flask api-et så hadde vi eget kall for mine `/notes/mine`.

Skal vi vente til rettigheter har blitt implementert?

Reply:
Jepp, det kan vi gjøre